### PR TITLE
add series interface of publics

### PR DESCRIPTION
**Pull request by VictorZhou123**
1. fix addlikes bugs
2. add field "owner" to wukongPictureItem
3. add new interface
   1. addPublicPicture
   2. ListPublicPicture
   3. GetPublicPictureGlobal

### CODE REVIEW

File: infrastructure/mongodb/wukong_picture.go [hunk 1]
```diff
@@ -240,13 +267,38 @@ func (col wukongPicture) toPictureDO(p *pictureItem, do *repositories.WuKongPict
...
 		OBSPath:   do.OBSPath,
 		Diggs:     do.Diggs,
 		DiggCount: do.DiggCount,
+		Version:   do.Version,
 		CreatedAt: do.CreatedAt,
 	})
+	if err != nil {
+		return nil, err
+	}
+	doc["Version"] = do.Version
```
Comment (zengchen1024) [inline infrastructure/mongodb/wukong_picture.go:284]: it is "version" not "Version". use fieldVersion instead directly.

File: infrastructure/mongodb/wukong_picture.go [hunk 2]
```diff
@@ -149,7 +156,6 @@ func (col wukongPicture) insert(
 	if err != nil {
 		return
 	}
-
```
Comment (zengchen1024) [inline infrastructure/mongodb/wukong_picture.go:152]: keep the blank line.

File: infrastructure/mongodb/doc.go [hunk 3]
```diff
@@ -493,34 +493,35 @@ type luojiaItem struct {
 }
 
 type dWuKong struct {
-	Id      string    `bson:"id"        json:"id"`
-	Samples []dSample `bson:"samples"   json:"samples"`
+    Id      string    `bson:"id"      json:"id"`
```
[the remark this answers is not in the document]
Comment (VictorZhou123) [reply infrastructure/mongodb/doc.go:496]: yes, I replaced it

File: infrastructure/mongodb/wukong_picture.go (hunk 1 above)
Comment (VictorZhou123) [reply infrastructure/mongodb/wukong_picture.go:284]: fixed @zengchen1024

File: infrastructure/mongodb/wukong_picture.go (hunk 2 above)
Comment (VictorZhou123) [reply infrastructure/mongodb/wukong_picture.go:152]: fixed

**Pull request merged**